packages/components/addon/components/hds/flyout/index.js

```diff
@@ -98,6 +104,9 @@ export default class HdsFlyoutIndexComponent extends Component {
     this.element.showModal();
     this.isOpen = true;
 
+    // Prevent page from scrolling when the dialog is open
+    if (this.body) this.body.style.overflow = 'hidden';
```

Comment:
I _think_ setting a style property directly like this might be incompatible with apps that use a strict Content Security Policy (this would violate inline-style directives). `[style.setProperty](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty)` safely updates the CSSOM (this is the API `ember-style-modifier` uses to safely add inline styles).

Reply:
Indeed, that's something worth investigating. I guess an alternative would be to add/remove a CSS class on the `<body>` element – not elegant either, but more in line with stricter CSPs

Reply:
ah, right, setters and getters are fine – thanks for the heads-up, will update and re-test